### ISSUE TITLE
Accessibility: Icon missing the alt attribute to compliant with IBM DAP a11y tool

### ISSUE THREAD

**Issue by lilyclyc** [carbon-design-system/carbon-components-react#162]
![image](https://user-images.githubusercontent.com/30255650/28914713-4e5c5426-786f-11e7-9d65-b55a1f33f7d5.png)
IBM DAP plugin will report an a11y issue about missing alt on svg, even it already has an aria-label.
As our project must compliant with IBM A11y policy, so would you work on adding the alt attribute for Icon component?

**Comment by hellobrian** [comment]
We chose not to use `alt` because it is not a valid attribute for SVG

**Comment by hellobrian** [comment]
If you want to add `alt` yourself, you can. All of our components including `Icon` accept any user-provided props via `...other` ([see here](https://github.com/carbon-design-system/carbon-components-react/blob/master/components/Icon.js#L102))

**Comment by lilyclyc** [comment]
@hellobrian -- Such as the icon in dropdown, there is no way to set alt on internal icon.

**Comment by hellobrian** [comment]
Unfortunately, the IBM DAP tool is out of date or not correct about it's understanding of inline `<svg>` accessibility.  

The current spec says that inline `<svg>` should provide a `<title>` to properly describe the image
But as browsers are slow to implement the standard, the recommended technique for screen readers is to require the combination of `role="img"` and `aria-label` in order to be accessible.
